lib/adapters/mongodb.js

```diff
@@ -1,6 +1,9 @@
 var mongoose = require('mongoose');
 var RSVP = require('rsvp');
```

Comment:
Since harvesterjs is switching to using bluebird for promises. Can we also include a quick refactor, switching out RSVP for bluebird within this file for this PR.